src/resources/views/base/inc/alerts.blade.php

```diff
@@ -2,7 +2,7 @@
 <script type="text/javascript">
     // This is intentionaly run after dom loads so this way we can avoid showing duplicate alerts
     // when the user is beeing redirected by persistent table, that happens before this event triggers.
-    window.addEventListener('DOMContentLoaded', function() {
```

Comment:
Quick question... would the problem still exist if we do `$( document ).ready()` instead of `window.addEventListener('DOMContentLoaded', function() {`? Does jQuery take care of this? Or CloudFlare work with that?

Reply:
@tabacitu as we discussed, I think jQuery can handle it properly, let's test it behind cloudflare, the Demo branch is here;
https://github.com/Laravel-Backpack/demo/pull/238
`fix-alerts-without-style`